styles.css

```diff
@@ -454,6 +454,207 @@ input[type=name] {
   align-items: center;
 }
 
+#work-section {
+  font-family: sans-serif;
+  font-weight: 500;
+  margin-top: 70px;
+}
+
+#box-1 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  margin-top: 40px;
+}
+
+#head {
+  border-bottom: 1px solid #646369;
+}
+
+#head h2 {
+  width: 50%;
+  font-size: 40px;
+  line-height: 54px;
+  color: #7d73fc;
+  margin-bottom: 30px;
+}
+
+#box-1 img {
+  width: 100%;
+  margin-bottom: 40px;
+  cursor: pointer;
+}
+
+#box-1 img:hover {
+  transform: scale(0.99);
+  transform: rotate(2deg);
+}
+
+#box-1 h1 {
+  font-family: sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin-bottom: 40px;
+}
+
+#box-1 p {
+  font-family: sans-serif;
+  font-weight: 400;
+  color: #dcdce8;
+  font-size: 20px;
+  line-height: 28px;
+  margin-bottom: 40px;
+}
+
+#box-1 ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  margin-bottom: 40px;
+}
+
+#box-2 ul li:nth-child(2),
+#box-2 ul li:nth-child(1),
+#box-2 ul li:nth-child(3) {
+  border-right: 1px solid #646369;
+  padding: 10px 20px;
+  font-family: sans-serif;
+  font-weight: 500;
+}
+
+#box-2 ul li:nth-child(4) {
+  padding: 10px 20px;
+  font-family: sans-serif;
+  font-weight: 500;
+}
+
+#button {
+  width: 20%;
+  font-family: sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 20px;
+  color: #fff;
+  background: #6a61d7;
+  border-radius: 1px;
+  padding: 12px 12px;
+  border: 1px solid #6a61d7;
+  transition: 0.2s;
+  cursor: pointer;
+}
+
+#button:hover {
+  background: #fff;
+  color: #6a61d7;
+}

```

Comment:
- [ ] Kindly update this to the correct hover styling provided by the design, the picture is provided above.